rafthttp/transport.go

```diff
@@ -87,6 +87,10 @@ type Transporter interface {
 	ActiveSince(id types.ID) time.Time
 	// Stop closes the connections and stops the transporter.
 	Stop()
+	// InitialPeerNotify returns a channel that closes when an initial
+	// peer connection has been established. Use this to wait until the
+	// first peer connection becomes active.
+	InitialPeerNotify() <-chan struct{}
```

Comment:
hmm... can you try to find another way to do this without introducing a new method to the interface? this interface is too heavy already.

Reply:
Agree. Just removed.